skietbaan_fe/src/components/ForgotPassword.css

```diff
@@ -30,15 +30,21 @@
   font-family: helvetica;
 }
 
+@media screen and (min-width: 575px){
+  .back-btn-forgot-password{
+    left: 20% !important;
```

Comment:
Is the !important necessary? One should always try and not use this

Reply:
to overwrite bootstrap